Makefile.in

```diff
@@ -48,7 +51,9 @@ clean:
 	$(RM) $(PROG) $(OBJECTS) $(MAN)
 
 distclean: clean
-	$(RM) Makefile config.h config.log config.cache config.status autom4te.cache config.h~ config.h.in~ config.h.in
+	$(RM) Makefile config.h config.log config.cache config.status autom4te.cache \
+	      config.h~ config.h.in~ config.h.in \
+	      $(DISTARCHIVE).tar.gz $(DISTARCHIVE).tar.bz2
```

Comment:
Will rm fail on nob existing files? Maybe -f?

Reply:
not*

Reply:
`autoconf` initializes `RM` as `/bin/rm -rf`, so this is already taken care of.